logging-operator/templates/cr.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.fluentbit.enabled }}
```

Comment:
please create separate cr for fluentbit and fluentd eg.: fluentbit-cr.yaml and fluentd-cr.yaml